api/index.js

```diff
@@ -6,12 +6,16 @@ const express = require('express')
 const swaggerizeExpress = require('swaggerize-express')
 const api = express()
 const rollbar = require('../config/rollbar')
+const bodyParser = require('body-parser')
```

Comment:
You're requiring `body-parser`, but it wasn't added as a dependency to the app.
`npm install --save body-parser` to add it to package.json